Client.xcodeproj/project.pbxproj

```diff
@@ -5159,7 +5155,6 @@
 				EB9A179B20E69A7F00B12184 /* ThemeManager.swift in Sources */,
 				EBFDB790211C83A5005CCA2F /* BrowserViewController+FindInPage.swift in Sources */,
 				EBB89509219398E500EB91A0 /* TabContentBlocker+ContentScript.swift in Sources */,
-				0B3E7D951B27A7CE00E2E84D /* AboutHomeHandler.swift in Sources */,
 				D3BE7B461B054F8600641031 /* TestAppDelegate.swift in Sources */,
```

Comment:
I guess this should not be removed...

Reply:
I think these were removed when I deleted the "class HomePageUITest: BaseTestCase" from the Project Navigator. Not sure how this can be reverted.

Reply:
Open the file where those lines are and add them in the same line. Then be sure they have the same indentation (check it with git diff)